packages/aws-client/src/Lambda.ts

```diff
@@ -1,26 +1,21 @@
-const Logger = require('@cumulus/logger');
-
-const awsServices = require('./services');
-const { inTestMode } = require('./test-utils');
+import Logger = require('@cumulus/logger');
+import { lambda } from './services';
+import { inTestMode } from './test-utils';
 
 const log = new Logger({ sender: 'aws-client/Lambda' });
 
-const invoke = async (name, payload, type = 'Event') => {
+export const invoke = async (name: string, payload: unknown, type = 'Event') => {
```

Comment:
Why not `any` instead of `unknown`?

Reply:
This explains it better than I could: https://mariusschulz.com/blog/the-unknown-type-in-typescript